GitHub/scripts/setup-runner.sh

```diff
@@ -51,7 +59,15 @@ mkdir $deploy_dir
 curl -o $deploy_dir/actions-runner.tar.gz -L https://github.com/actions/runner/releases/download/v$version/actions-runner-osx-x64-$version.tar.gz
 cd $deploy_dir && tar xzf $deploy_dir/actions-runner.tar.gz
 
-$deploy_dir/config.sh --url $repository --token $github_token --name $runner --work $work_dir --runnergroup $group --labels $labels
+if [[ $ephemeral = "false" ]]
+then
+    $deploy_dir/config.sh --url $repository --token $github_token --name $runner --work $work_dir --runnergroup $group --labels $labels
+elif [[ $ephemeral = "true" ]]
+then
+    $deploy_dir/config.sh --url $repository --token $github_token --name $runner --work $work_dir --runnergroup $group --labels $labels --ephemeral
+else
+    echo "Invalid input for the ephemeral tag."
+fi
```

Comment:
I think we can handle this more gracefully by passing the `$ephemeral` variable through the `--ephemeral` argument regardless of whether it is assigned a value by the user. If we assume the default value of `false`, we don't have to wrap the entry point of `config.sh` an an if/else statement.